static/src/images/commercial/masterclasses-logo-colour.svg

```diff
@@ -0,0 +1,56 @@
+	c0.354,0,0.883,0,1.147,0.088V5.693h-0.706c-1.456,0-2.647,0.883-3.53,2.736H77.19V5.561h-0.441L71.32,6.444L71.364,6.399z
+	 M88.93,19.816c2.205,0,4.104-1.061,4.854-1.942v-1.235c-0.661,0.176-2.073,0.44-3.311,0.44c-2.205,0-3.884-1.412-3.884-4.634
+	c0-3.708,1.147-5.12,2.647-5.12c0.617,0,1.104,0.089,1.324,0.222l0.75,2.648h2.34V6.09c-0.75-0.309-2.735-0.574-4.282-0.574
+	c-4.501,0-7.412,2.647-7.412,7.149c0,4.28,2.383,7.062,6.974,7.062V19.816z M94.488,0.838v1.324l1.5,0.574v14.741l-1.323,0.707
+	v1.322h7.416v-1.322l-1.457-0.619V0h-0.353l-5.738,0.883L94.488,0.838z M106.582,7.415h1.146c1.768,0,2.295,0.882,2.295,2.559v1.766
+	l-2.646,0.441c-2.957,0.573-4.768,1.104-4.768,3.973c0,2.205,1.5,3.75,3.884,3.75c1.767,0,3.312-0.75,3.973-1.939h0.089
+	c0.223,1.455,1.457,1.896,2.869,1.896c1.102,0,2.118-0.311,2.559-0.529v-1.018l-1.457-0.705V9.666c0-3.001-2.205-4.06-5.824-4.06
+	c-2.34,0-3.883,0.617-5.162,1.059v3.531h2.206l0.883-2.781H106.582z M110.023,16.902c-0.265,0.266-0.75,0.531-1.233,0.531
+	c-0.886,0-1.767-0.443-1.767-1.986c0-1.324,0.881-1.986,1.941-2.119l1.059-0.133v3.662V16.902z M121.057,19.859
+	c4.414,0,6.18-1.941,6.18-4.768c0-2.117-0.973-3.839-3.619-4.412l-1.588-0.353c-1.502-0.31-1.897-0.795-1.897-1.589
+	c0-0.883,0.75-1.324,1.767-1.324c0.795,0,1.764,0.176,2.029,0.309l0.573,1.897h1.896V6.267c-1.103-0.441-2.735-0.706-4.322-0.706
+	c-3.973,0-5.738,1.897-5.738,4.501c0,2.029,1.146,3.973,3.619,4.59l1.677,0.353c1.456,0.312,1.767,0.706,1.767,1.767
+	c0,1.016-0.883,1.324-2.207,1.324c-0.752,0-1.854-0.178-2.207-0.441l-0.705-2.207h-1.942v3.529c1.323,0.574,3.267,0.885,4.767,0.885
+	h-0.047V19.859z M132.666,19.859c4.413,0,6.178-1.941,6.178-4.768c0-2.117-1.015-3.839-3.619-4.412l-1.588-0.353
+	c-1.5-0.31-1.898-0.795-1.898-1.589c0-0.883,0.707-1.324,1.766-1.324c0.707,0,1.678,0.176,1.986,0.309l0.574,1.897h1.897V6.267
+	c-1.017-0.441-2.647-0.706-4.239-0.706c-3.971,0-5.734,1.897-5.734,4.501c0,2.029,1.103,3.973,3.617,4.59l1.635,0.353
+	c1.457,0.312,1.764,0.706,1.764,1.767c0,1.016-0.881,1.324-2.117,1.324c-0.795,0-1.898-0.178-2.205-0.441l-0.797-2.207h-1.896v3.529
+	c1.324,0.574,3.266,0.885,4.723,0.885h-0.045V19.859z M152.259,13.152c0-5.738-1.985-7.503-6.046-7.503s-6.708,2.56-6.708,7.062
+	c0,4.635,2.384,7.191,7.062,7.191c2.472,0,4.545-1.103,5.164-1.984v-1.324c-0.885,0.311-1.896,0.44-3.664,0.44
+	c-2.517,0-3.752-1.233-3.971-3.882h8.119H152.259z M146.081,7.105c1.147,0,1.767,0.574,1.767,4.192l-3.752,0.221
+	C144.096,7.9,144.844,7.105,146.081,7.105z M157.645,19.902c4.324,0,6.09-1.984,6.09-4.854c0-2.028-0.971-3.75-3.619-4.323
+	l-1.588-0.354c-1.502-0.309-1.854-0.794-1.854-1.589c0-0.882,0.707-1.324,1.767-1.324c0.75,0,1.677,0.177,1.986,0.309l0.573,1.898
+	h1.897V6.267c-1.104-0.441-2.737-0.706-4.324-0.706c-3.973,0-5.739,1.897-5.739,4.501c0,2.029,1.149,3.973,3.621,4.59l1.675,0.353
+	c1.457,0.312,1.768,0.706,1.768,1.767c0,1.016-0.883,1.324-2.207,1.324c-0.75,0-1.766-0.178-2.207-0.441l-0.707-2.207h-1.941v3.529
+	c1.324,0.574,3.311,0.885,4.768,0.885L157.645,19.902z"/>
+</svg>

```

Comment:
Smaller as live text ;-)

Bookshop, holidayoffers had _crazy_ viewBoxes that were cutting stuff off while viewing in a browser. I hope they weren't carefully positioned, because now they would have to be positioned all over again :-)

Good webapp for SVGO is @jakearchibald's SVGOMG. I still tend to prefer Illustrator when reducing no. of decimal places, though...

Can drop some more stuff (viewBox, namespace depending on usage)

bookshop:
```<svg xmlns="http://www.w3.org/2000/svg" width="117" height="26" viewBox="51 23.5 117 26" enable-background="new 51 23.5 117 26"><path fill="#7D0068" d="M111.5 42.7v.4H106l-3.5-5.7h-.7v3.8l1.5.7v1.2h-7.4V42l1.4-.7V26.5l-1.5-.7v-1.2l5.7-.9h.3v11.9h1l3.7-4.2-1.5-.7v-1.2h5.7v1.3l-1.8.6-3.1 3.4 4.1 6.4 1.5.7v-2.8h1.9l.8 2.2c.4.4 1.4.4 2.2.4 1.3 0 2.2-.2 2.2-1.3 0-1-.4-1.3-1.8-1.8l-1.8-.3c-2.4-.6-3.6-2.6-3.6-4.5 0-2.6 1.8-4.5 5.7-4.5 1.6 0 3.3.3 4.3.7v3.4h-1.9l-.6-1.9c-.3-.2-1.2-.4-2-.4-1.1 0-1.8.5-1.8 1.3 0 .9.3 1.3 1.8 1.6l1.6.4c2.7.6 3.6 2.3 3.6 4.4 0 2.8-1.9 4.8-6.2 4.8-1.2 0-3.1-.3-4.3-.9zM57 23.8h-.4l-5.6.8v1.2l1.5.7v15.8c1.2.7 3.4 1.2 5.9 1.2 4.2 0 7.3-1.8 7.3-7.4 0-5-2.4-6.9-4.7-6.9-1.8 0-3.1.8-3.8 1.6H57v-7zm1.5 18.1c-.8 0-1.2-.3-1.5-.4v-9.3c.4-.2.9-.4 1.6-.4 1.3 0 2.4 1 2.4 4.6 0 3.9-.8 5.5-2.5 5.5zm15-12.7c-4.2 0-7.1 2.6-7.1 7.2s2.9 7.2 7.1 7.2c4.1 0 6.9-2.5 6.9-7.2.1-4.7-2.8-7.2-6.9-7.2zm0 12.3c-1.5 0-2.2-.7-2.2-5.2S72 31 73.5 31c1.5 0 2.2.8 2.2 5.3s-.7 5.2-2.2 5.2zm14.6-12.3c-4.1 0-6.9 2.6-6.9 7.2s2.8 7.2 7 7.2c4.1 0 6.9-2.5 6.9-7.2s-2.8-7.2-7-7.2zm.1 12.3c-1.5 0-2.2-.7-2.2-5.2s.7-5.3 2.2-5.3c1.5 0 2.2.8 2.2 5.3s-.7 5.2-2.2 5.2zm42.1 1.7v-1.3l-1.5-.7v-8.8c.6-.2 1.2-.4 1.9-.4 1.2 0 1.8.5 1.8 1.9v7.4l-1.4.7v1.3h7.4V42l-1.5-.7v-8.4c0-2.5-1.1-3.7-3.4-3.7-1.8 0-3.5.7-4.7 1.8h-.2v-7.5h-.4l-5.6.9v1.1l1.5.7v15l-1.5.7v1.2l7.6.1zm15.6-14.1c-4.1 0-6.9 2.6-6.9 7.2s2.8 7.2 6.9 7.2c4.2 0 7-2.5 7-7.2 0-4.6-2.8-7.2-7-7.2zm.1 12.5c-1.5 0-2.2-.8-2.2-5.3s.8-5.3 2.2-5.3c1.5 0 2.2.8 2.2 5.3-.1 4.5-.7 5.3-2.2 5.3zm13.4-10.7v-1.8h-.4l-5.6.9v1.2l1.5.7v15.2l-1.5.7v1.3h7.6v-1.3l-1.7-.7v-4c.4.1.9.4 2 .4 3.5 0 6.6-1.9 6.6-7.4 0-5.3-2.2-6.9-4.9-6.9-1.8 0-2.9.9-3.7 1.8h-.2l.3-.1zm1.6 10.6c-.6 0-1.1-.2-1.4-.4v-8.8c.4-.2.8-.4 1.6-.4 1.5 0 2.3 1.3 2.3 4.5 0 3.8-1.1 5.1-2.5 5.1z"/></svg>```

holidayoffers:
```<svg xmlns="http://www.w3.org/2000/svg" width="152" height="25.5" viewBox="0 0 152 25.5"><path fill="#333" d="M.1 19.4h7.3v-1.2L6 17.5V8.7c.5-.2 1.1-.4 1.8-.4 1.2 0 1.8.5 1.8 1.9v7.3l-1.4.7v1.2h7.3v-1.2l-1.5-.7V9.3c0-2.5-1.1-3.6-3.3-3.6-1.8 0-3.5.7-4.7 1.8h-.1V0h-.4L0 .9v1.2l1.5.7v14.8l-1.5.7v1.1h.1zM22.8 5.7c-4.1 0-6.9 2.5-6.9 7s2.8 7 6.9 7c4.1 0 6.9-2.5 6.9-7s-2.8-7-6.9-7zm.1 12.2c-1.5 0-2.2-.8-2.2-5.2s.7-5.2 2.2-5.2 2.1.8 2.1 5.2-.7 5.2-2.1 5.2zm7.4-16.8v1.2l1.5.7v14.5l-1.4.7v1.2h7.3v-1.2l-1.4-.7V.2h-.4l-5.6.9zm7.8 5.4v1.2l1.5.7v9.1l-1.4.7v1.2h7.3v-1.2l-1.5-.7V5.7h-.4l-5.5.8zM41.8.2c-1.3 0-2.4 1.1-2.4 2.4 0 1.3 1.1 2.3 2.4 2.3 1.3 0 2.4-1.1 2.4-2.3C44.1 1.2 43 .2 41.8.2zm12.8 6.1c-.5-.4-1.4-.5-2.1-.5-3.5 0-6.5 1.9-6.5 7.3 0 5.1 2.9 6.7 5.4 6.7 1.7 0 2.8-.7 3.4-1.5h.1v1.4h.4l5.3-.6V18l-1.4-.8V.3h-.4l-5.6.9v1.2l1.4.6v3.3zm0 10.8c-.3.1-.7.2-1.4.2-1.8 0-2.6-1.4-2.6-4.7 0-3.8 1.1-5 2.5-5 .8 0 1.2.2 1.5.5v9zm13.5-.3c-.2.3-.7.5-1.2.5-.9 0-1.7-.5-1.7-2 0-1.3.8-2 1.9-2.1l1-.1v3.7zm-3.3-9.3h1.1c1.7 0 2.2.9 2.2 2.6v1.8l-2.6.4c-3 .5-4.7 1.1-4.7 3.8 0 2.2 1.5 3.8 3.8 3.8 1.7 0 3.2-.7 3.8-1.9h.1c.2 1.4 1.5 1.9 2.8 1.9 1.1 0 2.1-.3 2.5-.5v-1l-1.4-.7V9.8c0-3-2.1-4.1-5.7-4.1-2.3 0-3.8.6-5.1 1v3.5h2.1l1.1-2.7zm11.8 15.9c-.5 0-1.2 0-1.7-.2v1.6c.6.4 1.3.7 2.2.7 2.2 0 3.7-1 4.7-3.9l5-13.6 1.3-.7V6H83v1.2l1.4.6-2.3 7H82l-2.5-7 1.5-.5V6h-7.7v1.2l1.3.8L80 20.6l-.2.5c-.7 1.6-1.7 2.3-3.2 2.3zM94.7 5.7c-4.1 0-6.9 2.5-6.9 7s2.8 7 6.9 7c4.1 0 6.9-2.5 6.9-7s-2.8-7-6.9-7zm.1 12.2c-1.5 0-2.2-.8-2.2-5.2s.7-5.2 2.2-5.2 2.2.8 2.2 5.2c-.1 4.4-.8 5.2-2.2 5.2zm20.8-.4V8.3h2.2V6h-2.2V4.5c0-1.6.4-2.3 1.6-2.3h.5l.5 1.9h2.3V.7c-1.5-.4-2.5-.5-3.8-.5-1.4 0-2.8.4-3.8 1.2-1.2-.6-2.5-.7-3.7-.7-2.8 0-5.5 1.5-5.5 4.8v.6l-1.5.8v1.5h1.5v9.2l-1.5.7v1.2h7.2v-1.2l-1.3-.7V8.3h3.1v9.2l-1 .7v1.2h7.3v-1.2l-1.9-.7zM108.1 5c0-1.6.4-2.3 1.6-2.3.8 0 1.3.4 1.7 1.1-.1.4-.1.8-.1 1.3v1h-3.1l-.1-1.1zm16.4 2.2c1.1 0 1.7.6 1.7 4.1l-3.7.2c0-3.5.8-4.3 2-4.3zm6 5.9c0-5.6-1.9-7.4-5.9-7.4s-6.6 2.6-6.6 7c0 4.5 2.4 7.1 7 7.1 2.5 0 4.5-1.1 5.1-2v-1.3c-.9.3-1.9.5-3.6.5-2.4 0-3.7-1.2-3.9-3.8l7.9-.1zm.4-6.6v1.2l1.5.7v9.1l-1.4.7v1.2h7.7v-1.2l-1.8-.7v-7.4c.8-.4 1.8-.5 3-.5.4 0 .9 0 1.1.1V5.8c-.1 0-.4-.1-.7-.1-1.5 0-2.6 1-3.4 2.7h-.1V5.7h-.4l-5.5.8zM146 19.8c4.2 0 6-1.9 6-4.7 0-2-1-3.7-3.6-4.3l-1.6-.3c-1.5-.3-1.8-.8-1.8-1.6s.7-1.3 1.8-1.3c.7 0 1.6.2 2 .3l.6 1.8h1.9V6.4c-1.1-.5-2.7-.7-4.2-.7-3.9 0-5.7 1.9-5.7 4.4 0 2 1.1 3.9 3.6 4.5l1.6.4c1.4.3 1.8.7 1.8 1.7 0 1.1-.9 1.3-2.1 1.3-.8 0-1.8-.1-2.1-.4l-.8-2.2h-1.9v3.5c1.1.6 3.1.9 4.5.9z"/></svg>```

jobs:
```<svg xmlns="http://www.w3.org/2000/svg" width="51" height="26" viewBox="0 0 51 26"><path fill="#197CAA" d="M3.4 8.4v13.5c0 1.1-1 1.8-2.1 1.8-.4 0-.9 0-1.3-.1v1.6c.6.4 1.7.6 2.8.6 3.4 0 5-1.8 5-4.9V5.6h-.2l-5.8.8v1.3l1.6.7zM5.6 0C4.2 0 3.1 1.1 3.1 2.4c0 1.3 1.1 2.4 2.4 2.4C6.9 4.8 8 3.7 8 2.4 8 1.1 6.9 0 5.6 0zm10.8 5.7c-4.1 0-6.9 2.5-6.9 7.1 0 4.7 2.8 7.2 7 7.2s7-2.6 7-7.2-3-7.1-7.1-7.1zm0 12.3c-1.5 0-2.2-.8-2.2-5.3 0-4.3.8-5.2 2.2-5.2 1.5 0 2.2.8 2.2 5.3 0 4.4-.7 5.3-2.2 5.2zM29.8.1h-.4l-5.6.9v1.3l1.4.5v15.8c1.2.7 3.4 1.2 6 1.2 4.2 0 7.4-1.9 7.4-7.4 0-5-2.5-6.9-4.9-6.9-1.7 0-3 .8-3.8 1.5h-.2l.1-6.9zm1.5 18.3c-.8 0-1.2-.3-1.5-.4V8.7c.4-.2.9-.3 1.6-.3 1.4 0 2.4.9 2.4 4.6.1 3.8-.8 5.4-2.5 5.4zM44.1 20c4.3 0 6.1-1.9 6.1-4.7 0-2.1-1-3.8-3.6-4.4l-1.6-.4c-1.5-.3-1.9-.8-1.9-1.6 0-.9.7-1.3 1.8-1.3.8 0 1.7.1 2 .3l.6 1.9h1.9V6.4c-1.1-.4-2.7-.7-4.3-.7-4.1 0-5.8 1.9-5.8 4.5 0 2 1.2 4 3.7 4.6l1.8.4c1.4.3 1.8.7 1.8 1.7 0 1.1-.9 1.3-2.2 1.3-.8 0-1.9-.1-2.2-.4l-.8-2.2h-1.9v3.6c1.2.5 3.1.9 4.6.8z"/></svg>```

masterclass:
```<svg xmlns="http://www.w3.org/2000/svg" width="164" height="20" viewBox="0 0 164 20"><path fill="#333" d="M.1 19.5h7.2v-1.2L6 17.6v-9c.5-.2 1-.3 1.5-.3 1.2 0 1.7.4 1.7 1.9v7.3l-1.2.7v1.3h6.9v-1.2l-1.2-.7v-9c.4-.2 1-.3 1.5-.3 1.2 0 1.8.4 1.8 1.9v7.3l-1.3.8v1.2h7.1v-1.2l-1.4-.7V9.2c0-2.5-1.1-3.6-3.4-3.6-1.9 0-3.2.7-4.4 1.8h-.1C13 6.1 12 5.6 10.4 5.6c-1.9 0-3 .7-4.2 1.8H6V5.6h-.4L0 6.4v1.1l1.5.8v9.3l-1.5.5v1.4zM27.5 7.4h1.1c1.8 0 2.2.9 2.2 2.6v1.8l-2.6.4c-3 .6-4.9 1.1-4.9 4 0 2.2 1.5 3.8 4 3.8 1.7 0 3.3-.8 3.9-1.9h.1c.2 1.5 1.5 1.9 2.9 1.9 1.1 0 2.1-.3 2.5-.5v-1l-1.5-.7V9.6c0-3-2.2-4.1-5.8-4.1-2.4 0-4 .6-5.2 1.1v3.5h2.1l.9-2.8.3.1zm3.3 9.5c-.2.3-.8.5-1.2.5-.9 0-1.7-.4-1.7-2 0-1.3.9-2 1.9-2.1l1.1-.1v3.7zm11.1 2.9c4.3 0 6.2-1.9 6.2-4.8 0-2.1-1.1-3.8-3.7-4.4l-1.6-.4C41.3 10 41 9.5 41 8.7c0-.9.7-1.3 1.8-1.3.8 0 1.8.2 2 .3l.6 1.9h1.9V6.3c-1.1-.4-2.6-.7-4.3-.7-4 0-5.7 1.9-5.7 4.5 0 2 1.1 4 3.7 4.6l1.7.4c1.5.3 1.8.7 1.8 1.8 0 1-.9 1.3-2.2 1.3-.7 0-1.8-.2-2.1-.4l-.8-2.2h-1.9V19c1 .5 3 .9 4.4.8zm8.3-4c0 2.9 1.3 4.1 4 4.1 1.4 0 2.9-.4 3.7-.8v-1.8l-1.5.1c-1.3 0-1.7-.7-1.7-2V8.2h3V5.9h-3V2.7l-4.5.6v2.6l-2.1.3v2h2.1v7.6zm20.7-2.7c0-5.7-1.9-7.5-6-7.5s-6.7 2.6-6.7 7.1c0 4.6 2.4 7.2 7.1 7.2 2.5 0 4.5-1.1 5.2-2v-1.3c-1 .3-1.9.4-3.7.4-2.5 0-3.8-1.2-4-3.9h8.1zm-6.1-6c1.1 0 1.8.6 1.8 4.2l-3.8.2c0-3.6.8-4.4 2-4.4zm6.6-.7v1.2l1.5.8v9.3l-1.4.6v1.3h7.8v-1.3l-1.9-.6v-7.5c.8-.4 1.8-.5 3-.5.4 0 .9 0 1.1.1v-4h-.7c-1.5 0-2.6.9-3.5 2.7h-.1V5.6h-.4l-5.4.8zm17.5 13.4c2.2 0 4.1-1.1 4.9-1.9v-1.2c-.7.2-2.1.4-3.3.4-2.2 0-3.9-1.4-3.9-4.6 0-3.7 1.1-5.1 2.6-5.1.6 0 1.1.1 1.3.2l.8 2.6h2.3V6.1c-.8-.3-2.7-.6-4.3-.6-4.5 0-7.4 2.6-7.4 7.1.1 4.3 2.4 7.1 7 7.2zm5.6-19v1.3l1.5.6v14.7l-1.3.7v1.3h7.4v-1.3l-1.5-.6V0h-.4l-5.7.8zm12.1 6.6h1.1c1.8 0 2.3.9 2.3 2.6v1.8l-2.6.4c-3 .6-4.8 1.1-4.8 4 0 2.2 1.5 3.8 3.9 3.8 1.8 0 3.3-.8 4-1.9h.1c.2 1.5 1.5 1.9 2.9 1.9 1.1 0 2.1-.3 2.6-.5v-1l-1.5-.7V9.7c0-3-2.2-4.1-5.8-4.1-2.3 0-3.9.6-5.2 1.1v3.5h2.2l.8-2.8zm3.4 9.5c-.3.3-.8.5-1.2.5-.9 0-1.8-.4-1.8-2 0-1.3.9-2 1.9-2.1l1.1-.1v3.7zm11.1 3c4.4 0 6.2-1.9 6.2-4.8 0-2.1-1-3.8-3.6-4.4l-1.6-.4c-1.5-.3-1.9-.8-1.9-1.6 0-.9.8-1.3 1.8-1.3.8 0 1.8.2 2 .3l.6 1.9h1.9V6.3c-1.1-.4-2.7-.7-4.3-.7-4 0-5.7 1.9-5.7 4.5 0 2 1.1 4 3.6 4.6l1.7.4c1.5.3 1.8.7 1.8 1.8 0 1-.9 1.3-2.2 1.3-.8 0-1.9-.2-2.2-.4l-.7-2.2h-1.9V19c1.1.6 3 .9 4.5.9zm11.6 0c4.4 0 6.2-1.9 6.2-4.8 0-2.1-1-3.8-3.6-4.4l-1.6-.4c-1.5-.3-1.9-.8-1.9-1.6 0-.9.7-1.3 1.8-1.3.7 0 1.7.2 2 .3l.6 1.9h1.9V6.3c-1-.4-2.6-.7-4.2-.7-4 0-5.7 1.9-5.7 4.5 0 2 1.1 4 3.6 4.6l1.6.4c1.5.3 1.8.7 1.8 1.8 0 1-.9 1.3-2.1 1.3-.8 0-1.9-.2-2.2-.4l-.8-2.2H128V19c1.3.6 3.3.9 4.7.9zm19.6-6.7c0-5.7-2-7.5-6-7.5s-6.7 2.6-6.7 7.1c0 4.6 2.4 7.2 7.1 7.2 2.5 0 4.5-1.1 5.2-2v-1.3c-.9.3-1.9.4-3.7.4-2.5 0-3.8-1.2-4-3.9h8.1zm-6.2-6.1c1.1 0 1.8.6 1.8 4.2l-3.8.2c0-3.6.7-4.4 2-4.4zm11.5 12.8c4.3 0 6.1-2 6.1-4.9 0-2-1-3.8-3.6-4.3l-1.6-.4c-1.5-.3-1.9-.8-1.9-1.6 0-.9.7-1.3 1.8-1.3.8 0 1.7.2 2 .3l.6 1.9h1.9V6.3c-1.1-.4-2.7-.7-4.3-.7-4 0-5.7 1.9-5.7 4.5 0 2 1.1 4 3.6 4.6l1.7.4c1.5.3 1.8.7 1.8 1.8 0 1-.9 1.3-2.2 1.3-.8 0-1.8-.2-2.2-.4l-.7-2.2H153V19c1.2.6 3.1.9 4.6.9z"/></svg>```

Regards
Mateusz